components/operationModal/index.js

```diff
@@ -0,0 +1,26 @@
+import TransactionOperation from '@components/transaction-operation';
+import React, { useEffect, useState } from 'react';
+import receivers from '../../mock/receivers';
+const OperationModal = (props) => {
+  const { transactionType, personData } = props;
```

Comment:
NIT : the destructuring can be done right on line no 4
```javascript
const OperationalModal = ({ transactionType, personData }) => {
```

Reply:
I was bit confused whether that is a good practise or writing it down, I will follow your approach from now onwards

Reply:
The approach you followed makes sense if you have like say 8-9 props coming in ..then for code readability, its better to destructure it in a new line .. but for 3-4 props, its better to destructure it like I mentioned :)